ROADMAP.md

```diff
@@ -0,0 +1,14 @@
+# Skopeo Roadmap
+
+The Skopeo development team reviews feature requests from its various stakeholders for consideration
+quarterly.  Skopeo maintainers then prioritize these features.   Top features are then assigned to
+one or more engineers.
```

Comment:
(Assigned? That seems hard to do in a volunteer-driven community project. \*shrug\*, if this is what we are saying everywhere, fine.)